app/components/layout/FindHeader.jsx

```diff
@@ -22,8 +22,17 @@ class FindHeader extends React.Component {
   render() {
     return (
       <header className="hero header-large" role="banner">
-        <h1>Hello, what can we help you find?</h1>
-        <h3>Search 1,138 housing and homelessness related services in San Francisco</h3>
+        <div className="header-container">
+          <div className="header-text">
+            <p>Welcome to the SF Service Guide</p>
+            <h1>Find food, housing, health<br />resources and more<br />in San Francisco</h1>
```

Comment:
These explicit `<br>`s might cause weird breaking on certain screen sizes, such as in the event where a screen size is small enough to wrap right before the word "health". Can we achieve the desired appearance by instead setting a maximum width on the `<h1>` or possibly the `<div className="header-text">`?